snuba/query/matchers.py

```diff
@@ -349,4 +350,36 @@ def match(self, node: AnyType) -> Optional[MatchResult]:
         return result
 
 
+@dataclass(frozen=True)
+class SubscriptableReference(Pattern[SubscriptableReferenceExpr]):
+    """
+    Matches a SubscriptableReference in the AST expression.
+    It works like the Column Pattern. if alias, column_name and key arguments
```

Comment:
Alias ? (there are no alias in the matchers, we ignore them)

Reply:
I copied this from FunctionCall, I'll change it.